awpy/types.py

```diff
@@ -1,9 +1,22 @@
 """This module contains the type definitions for the parsed json structure."""
 
 from dataclasses import dataclass
+<<<<<<< HEAD
 from typing import Literal, NewType, NotRequired, TypeGuard, TypeAlias, final, overload
 
 from typing_extensions import TypedDict
+=======
+from typing import (
+    Literal,
+    NotRequired,
+    TypeAlias,
+    TypedDict,
+    TypeGuard,
+    final,
+    overload,
+)
+
+>>>>>>> 8194355 (Changes to types.py after rebasing main branch updates)
```

Comment:
You left some merge conflict markers in.

Reply:
CI is failing because black crashes here.

Reply:
Yeah I forgot to change this on the latest commit. I fixed this locally but I have a few more commits to make before the PR will be ready for review again. Thanks for taking a look again and I'll let you know once its ready again.

Reply:
Addressed this issue in latest commit